lockbox-iosTests/BaseDataStoreSpec.swift

```diff
@@ -43,8 +43,9 @@ class BaseDataStoreSpec: QuickSpec {
             self.ensureLockedCalled = true
         }
 
-        func sync(unlockInfo: SyncUnlockInfo) throws {
+        func sync(unlockInfo: SyncUnlockInfo) throws -> String {
             self.syncArgument = unlockInfo
+            return ""
```

Comment:
Nit: prefer a string that documents what this is, rather than an empty string. 

e.g. 

```swift
return "fake-unlock-info"
```